sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -118,11 +113,11 @@ export class NotificationActionsService {
    * @param auditUserId user that should be considered the one that is causing the changes.
    * @param options options for the student restriction notification.
    */
-  async sendStudentRestrictionAddedNotification(
+  async saveStudentRestrictionAddedNotification(
     notifications: StudentRestrictionAddedNotification[],
     auditUserId: number,
     options?: StudentRestrictionAddedNotificationOptions,
```

Comment:
I believe that we can get rid of this class now and make this method the same as the others.